website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -59,7 +60,7 @@ var Range = ace.require('ace/range').Range;
         undo: "Undo -",
         redo: "Redo -",
 
-        help: "Markdown Editing Help"
+        help: "Wiki Syntax Help"

```

Comment:
As per osf-style, use sentence case for modal headers.

https://centerforopenscience.github.io/osf-style/language.html#capitalization